samples/storage_csm_unity_v270.yaml

```diff
@@ -43,6 +43,16 @@ spec:
           value: debug
         - name: TENANT_NAME
           value: ""
+        - name: CERT_SECRET_COUNT
```

Comment:
I think we should add a comment to this similar to the one for X_CSI_UNITY_SKIP_CERTIFICATE_VALIDATION documenting the default value of 1 etc.

Reply:
Added the comments for the same.